common/alter_operation_error.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+parent: Common
+---
+
+# Add Operation Error
+
+Understanding the `AlterOperationError` exception.
+{: .fs-6 .fw-300 }
+
+&nbsp;
+
+{: .warning }
+> _An **Add** operation targeting an existing profile could end up effectively performing an **Alter** operation on this profile. Pyracf will always raise an `AlterOperationError` to bring attention to these failures before attempting the **Add** operation._
```

Comment:
```markdown
> _An **Alter** operation targeting a profile that does **NOT** exist already could end up effectively performing an **Add** operation to create a new profile. Pyracf will always raise an `AlterOperationError` and refuse to perform the requested operation to bring attention to this condition._
```

Reply:
There were a lot of concerns here. I thought I had revised this file and apparently missed some sections. This has been remedied.